ecommers/lib/core/mixins/items_loading_notifier.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+import 'package:ecommers/core/app_services/index.dart';
+import 'package:ecommers/core/services/dependency_service.dart' as services;
+
+mixin ItemsLoadingNotifier on ChangeNotifier {
+  @protected
+  final Paginator paginator = services.paginator;
+
+  bool _hasDisposed = false;
+  bool _isitemsLoading = false;
+
+  bool get isitemsLoading => _isitemsLoading;
```

Comment:
Rename isitemsLoading  to isItemsLoading

Reply:
check this variable in other places

Reply:
corrected spelling